coil-core/api/android/coil-core.api

```diff
@@ -535,6 +578,25 @@ public final class coil3/request/ErrorResult : coil3/request/ImageResult {
 	public fun toString ()Ljava/lang/String;
 }
 
+public final class coil3/request/GifsKt {
```

Comment:
`ScaleDrawable`, `GifsKt`, `AnimatedTransformation`, and `PixelOpacity` should stay in the `coil-gif` artifact

Reply:
Done